build-system/common/utils.js

```diff
@@ -145,6 +145,10 @@ function getFilesToCheck(globs, options = {}, ignoreFile = undefined) {
       log(green('INFO: ') + 'No files to check in this PR');
       return [];
     }
+    // forbidden-terms always needs to be checked.
+    if (!filesChanged.includes('build-system/test-configs/forbidden-terms.js')) {
+      filesChanged.push('build-system/test-configs/forbidden-terms.js');
+    }
```

Comment:
Optional: Here's a more compact shorthand that might work. (Feel free to ignore if you don't like the syntax.)

```suggestion
    const forbiddenTerms = 'build-system/test-configs/forbidden-terms.js';
    filesChanged.includes(forbiddenTerms) && filesChanged.push(forbiddenTerms);
```